### PR TITLE
Fix CubeRed and Dual Core Bootloader issues

### PR DESCRIPTION
**Pull request by bugobliterator**
* Disable second core by default
* Fix bug in dual usb bootloader related to timeout.
* Rebuild bootloaders for dual core Cubes

### CODE REVIEW

File: Tools/ardupilotwaf/boards.py [hunk 1]
```diff
@@ -286,15 +286,15 @@ def configure_env(self, cfg, env):
 
         if cfg.env.DEBUG:
             env.CFLAGS += [
-                '-g',
+                '-g3',
```
Comment (tridge) [inline Tools/ardupilotwaf/boards.py:289]: check works on clang? or enable only on gcc

File: libraries/AP_HAL_ChibiOS/hwdef/common/board.c [hunk 2]
```diff
@@ -269,6 +269,30 @@ void __early_init(void) {
                      STM32_NOCACHE_MPU_REGION_2_SIZE |
                      MPU_RASR_ENABLE);
 #endif
+#if defined(DUAL_CORE)
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/common/board.c:272]: not critical, but prefer a fn in stm32_util.c

**Review by tridge — CHANGES_REQUESTED** [review]

File: Tools/ardupilotwaf/boards.py (hunk 1 above)
Comment (bugobliterator) [reply Tools/ardupilotwaf/boards.py:289]: --debug build added to CI check.

File: libraries/AP_HAL_ChibiOS/hwdef/common/board.c (hunk 2 above)
Comment (bugobliterator) [reply libraries/AP_HAL_ChibiOS/hwdef/common/board.c:272]: done

**Review by tridge — APPROVED** [review]

**Pull request merged**